worldedit-core/src/main/java/com/sk89q/worldedit/command/SchematicCommands.java

```diff
@@ -984,7 +1057,32 @@ public Void call() throws Exception {
             //FAWE end
             return null;
         }
+    }
+
+    private static class SchematicShareTask extends SchematicOutputTask<URL> {
+        private final String name;
 
+        SchematicShareTask(Actor actor, ClipboardFormat format, ClipboardHolder holder, String name) {
+            super(actor, format, holder);
+            this.name = name;
+        }
+
+        @Override
+        public URL call() throws Exception {
+            ByteArrayOutputStream baos = new ByteArrayOutputStream();
+            try {
+                writeToOutputStream(baos);
+            } catch (Exception e) {
+                throw new CommandException(TextComponent.of(e.getMessage()), e, ImmutableList.of());
+            }
+
+            EngineHubPaste pasteService = new EngineHubPaste();
```

Comment:
Is this something we're comfortable using in FAWE? We already have the schematic download portal ourselves

Reply:
If I remember me's blog entry correctly, other plugins should be able to hook into the paste system and allow custom schematic paste / load services. If that's still the case, our portal should be accessed via that api - the EngineHub way could still be enabled (iirc, that was even a command argument - tho I'm not seeing anything like that in this PR), if the EH peeps don't have any issue with that. That's totally up to them, and we shouldn't utilize their service if they're not fine with that.

Reply:
I don't have a strong opinion on that stuff, the main reason for this PR is that otherwise we have major merge conflicts for other stuff in 7.3 and in future.

Reply:
Probably worth just double checking first really

Reply:
I disabled the feature for now, so we can integrate the code and take care of details later